pkg/tuned/change-profile.jsx

```diff
@@ -34,17 +36,27 @@ const _ = cockpit.gettext;
  */
 class TunedDialogProfile extends React.Component {
     render() {
-        var classes = "list-group-item";
-        if (this.props.selected)
+        let classes = "list-group-item";
+        let variant = "filled";
+
+        if (this.props.selected) {
             classes += " active";
-        var recommended;
-        if (this.props.recommended)
-            recommended = <span className="badge pull-right">{ _("recommended") }</span>;
+            variant = "outline";
+        }
+
         return (
             <button className={ classes } key={ this.props.name } onClick={ this.props.click }>
```

Comment:
Oh that's so weird that it's a button here for the list rows. Nothing in the spectrum of this PR of course.

Reply:
yeah, it indeed is and I started to migrate it to something more reasonable, but then thought that could open up can of worms with style etc. so just kept it for some other time :)